src/app/views/cheat-sheets/mining/mining.component.html

```diff
@@ -3,7 +3,8 @@
         <div class="col-12 col-lg-6">
             <ul>
                 <li><a href="https://wiki.factorio.com/Mining" target="_blank" rel="noopener">Mining speed</a> relies on more variables than most other machinery.</li>
-                <li>Ores have <i>mining hardness</i> and <i>mining time</i>, and miners have <i>mining power</i> and <i>mining speed</i>.</li>
+                <li>0.17 has simplified the mining calculation for the base game.</li>
```

Comment:
I think we can get away without mentioning 0.17 changes specifically, after its standard no1 will care how it compares with previous versions

Reply:
Sounds good to me, will be removing the 0.17 line completely, and redoing the pull request to the 0.17 branch.